src/sentry/static/sentry/app/components/group/pluginActions.tsx

```diff
@@ -10,17 +11,33 @@ import {t} from 'app/locale';
 import SentryTypes from 'app/sentryTypes';
 import IssueSyncListElement from 'app/components/issueSyncListElement';
 import withOrganization from 'app/utils/withOrganization';
+import {Organization, Group, Project, Plugin} from 'app/types';
 
-class PluginActions extends React.Component {
-  static propTypes = {
+type Props = {
+  api: Client;
+  group: Group;
+  organization: Organization;
+  project: Project;
+  plugin: Plugin;
+};
+
+type State = {
+  showModal: boolean;
+  actionType: 'create' | 'link' | null;
```

Comment:
I would create a separate type for this, like for example:

type Action = 'create' | 'link'

if you do so you can reuse it in the `handleClick` function 😉

Reply:
Went with Billy's suggestion to handle this